core/templates/components/question-directives/modal-templates/question-editor-save-modal.component.html

```diff
@@ -6,7 +6,7 @@ <h3>
 <div class="modal-body oppia-long-text">
   <p>
     Please enter a brief description of what you have changed:
-    <textarea rows="3" cols="50" [attr.maxlength]="MAX_COMMIT_MESSAGE_LENGTH" [(ngModel)]="commitMessage" (focus)="saveChangesModalOpened" class="e2e-test-commit-message-input" autofocus></textarea>
+    <textarea rows="3" cols="50" [attr.maxlength]="MAX_COMMIT_MESSAGE_LENGTH" [(ngModel)]="commitMessage" class="e2e-test-commit-message-input" autofocus></textarea>
```

Comment:
`saveChangesModalOpened` is not even called and doesn't exist. Caught by angular compiler. It is really awesome.